src/Raven.Studio/typescript/commands/database/cmpXchg/saveCompareExchangeItemCommand.ts

```diff
@@ -2,9 +2,10 @@ import commandBase = require("commands/commandBase");
 import database = require("models/resources/database");
 import endpoints = require("endpoints");
 
-class saveCompareExchangeValueCommand extends commandBase {
+class saveCompareExchangeItemCommand extends commandBase {
 
-    constructor(private database: database, private key: string, private index: number, private data: any) {
+    constructor(private database: database, private key: string, private index: number,
+                private valueData: any, private metaData: any) {
```

Comment:
IIRC casing that we use for metadata is: metadata not metaData.

Reply:
done